var/spack/repos/builtin/packages/sirius/package.py

```diff
@@ -94,11 +94,11 @@ class Sirius(CMakePackage, CudaPackage):
     depends_on('magma', when='+magma')
     depends_on('boost cxxstd=14 +filesystem', when='+boost_filesystem')
 
-    depends_on('spfft', when='@6.4.0:')
+    depends_on('spfft@1.0.3', when='@6.4.0:')
```

Comment:
```suggestion
    depends_on('spfft@1.0.3:', when='@6.4.0:')
```
`1.0.3:` is for 1.0.3 and above (including 1.1.0, 2.0.0, ... infinity)

Reply:
fixed!